core-tests/src/test/java/org/mule/api/transport/ExceptionHandlingReplyToHandlerDecoratorTestCase.java

```diff
@@ -14,6 +14,7 @@
 import org.mule.api.MessagingException;
 import org.mule.api.MuleEvent;
 import org.mule.api.exception.MessagingExceptionHandler;
+import org.mule.execution.ExceptionHandlingReplyToHandlerDecorator;
```

Comment:
it was, but moved the test to a different package so I can remove the import. Also the package didn't make sense, was called transport.